saltstack/base/salt/performance/thp.sls

```diff
@@ -1,14 +1,17 @@
+{% if grains['init'] in [ 'upstart', 'sysvinit'] %}
 /etc/init.d/disable-thp:
   file.managed:
-    - source: salt://{{ slspath }}/etc/init.d/disable-thp
+    - source:
+      - salt://{{ slspath }}/etc/init.d/disable-thp.{{ grains['os'] | lower }}
```

Comment:
This only exists for debian and also seems to me that it does the same as [disable-thp](https://github.com/hortonworks/cloudbreak-images/blob/master/saltstack/base/salt/performance/etc/init.d/disable-thp) already does

Reply:
It is also installed for centos6 as it uses sysv init system. Debian7 has some kind of dependency management that is why BEGIN INIT INFO and END INIT INFO comments are there. As the 2 files are similar it is possible to merge them together.